src/components/atoms/Heading/Heading.tsx

```diff
@@ -3,6 +3,7 @@ import styled from 'styled-components';
 import Text from '../Text/Text';
 import { typography } from '../../../constants/typography';
 import { colors, THeadingHexColors } from '../../../constants/colors';
+import { maxMedia } from '../../../helpers/responsiveness';
```

Comment:
nitpicking but `minMedia` is generally preferred (mobile-first approach)

Reply:
Ha ha I thought so too, my bad habit 😅 , I'll update 👍

Reply:
I don't seem to get it working with `minMedia` for some reason 🤔 
```ts
 styled(Text)`
  font-size: ${({ as }) => (as === 'h1' ? '36px' : headingSizes[as])};
  ${({ as }) => minMedia.phone`font-size: ${headingSizes[as]};`};
`
```
Something related to having template literals within template literals? 🤯
The code ends much harder to read as well, so I'm tempted to leave it with `maxMedia` 😇